packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -300,6 +313,29 @@ public final class GeneratedPluginRegistrant {
 }
 ''';
 
+const String _androidPluginRegistryTemplateNewEmbedding = '''package dev.flutter.plugins;
+
+import io.flutter.embedding.engine.FlutterEngine;
+import io.flutter.embedding.engine.plugins.shim.ShimPluginRegistry;
+
+/**
+ * Generated file. Do not edit.
```

Comment:
Consider giving a pointer back to here and/or explaining how to generate it or where it came from.

Reply:
Done